ci/logstash_releases.json

```diff
@@ -1,12 +1,12 @@
 {
   "releases": {
     "7.current": "7.17.28",
-    "8.previous": "8.17.4",
+    "8.previous": "8.17.5",
     "8.current": "8.18.0"
   },
   "snapshots": {
     "7.current": "7.17.29-SNAPSHOT",
-    "8.previous": "8.17.5-SNAPSHOT",
+    "8.previous": "8.17.6-SNAPSHOT",
```

Comment:
let's kick the snapshot DRA off for 8.17

Reply:
Will do. thanks